commercial/app/controllers/commercial/ContentApiOffersController.scala

```diff
@@ -17,6 +17,12 @@ object ContentApiOffersController extends Controller with ExecutionContexts with
     "foundation-supported" -> "fc-container--foundation-supported"
   )
 
+  private val sponsorTypeToClassRefactor = Map(
```

Comment:
I like this typing, but rather than overriding `toString` maybe create a new property to make it more explicit what it's for.